composer.json

```diff
@@ -17,7 +17,7 @@
         "source": "https://github.com/php-tuf/composer-stager"
     },
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.4",
```

Comment:
You're also dropping PHP 8 support with this change. Surely this should be `>=7.4`?

Reply:
Yeah, I caught that after pushing. :bonk: It's fixed. Thanks. 🙂